app/js/actions/plugins.js

```diff
@@ -20,7 +64,8 @@ export const loadWorkflows = () => {
             .then((response) => (response.json()))
             .then((json) => {
                 Object.keys(json.workflows).map(workflow =>
-                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow]))
+                    dispatch(foundWorkflow(plugin.name, json.workflows[workflow])) &&
```

Comment:
Yeah, that was an eslint issue, where 

**Error [(array-callback-return)](http://eslint.org/docs/rules/array-callback-return) Expected to return a value in this function. at line ...**

I'll switch it to just return nothing.